### ISSUE TITLE
Is it possible to run SeamlessStreaming on an Apple M1 Pro?

### ISSUE THREAD

**Issue by snmishra311** [facebookresearch/seamless_communication#349]
I want to build a simple desktop app that translates a user's language in real-time.

Using Blackhole, I'd like to stream the audio from say a Zoom call into the SeamlessStreaming model and output the audio back into Zoom as a virtual Mic.

Is it possible to use SeamlessStreaming to do this locally on an M1 mac?

**Comment by mikestaub** [comment]
Yes, have it running on my M4 MacBook. You can use CPU, you don't need CUDA.
